facilities.csv

```diff
@@ -559,10 +565,10 @@ http://irail.be/stations/NMBS/008833159,Sint-Joris-Weert,Stationsstraat Zn.,3051
 http://irail.be/stations/NMBS/008822228,Sint-Katelijne-Waver,Stationsplein 1,2860,Sint-katelijne-waver,1,0,1,0,0,0,1,0,0,0,0,2,1,0,0,0,0,,,,,,,,,,,,,,
 http://irail.be/stations/NMBS/008821543,Sint-Mariaburg,Lodewijk Andessonstraat,2180,Antwerpen (ekeren),1,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,,,,,,,,,,,,,,
 http://irail.be/stations/NMBS/008812245,Sint-Martens-Bodegem,Stationsplein,1700,Sint Martens Bodegem - Dilbeek,1,0,1,0,0,0,0,0,0,0,0,0,1,0,0,0,0,,,,,,,,,,,,,,
-http://irail.be/stations/NMBS/008894508,Sint-Niklaas,Leopold Ii Laan 2,9100,Sint-niklaas,1,1,1,1,1,1,1,0,0,0,1,9,0,1,0,1,1,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,06:00,20:00,06:00,20:00
+http://irail.be/stations/NMBS/008894508,Sint-Niklaas,Leopold Ii Laan 2,9100,Sint-niklaas,1,0,1,1,1,1,1,0,0,0,1,9,0,1,0,1,1,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,05:45,20:00,06:00,20:00,06:00,20:00
```

Comment:
Sint-Niklaas instead of Sint-niklaas

Reply:
It probably should be Leopold II Laan

Reply:
@DylanVanAssche think the current casing is correct, according to wikipedia and the communal website https://nl.wikipedia.org/wiki/Sint-Niklaas
https://www.sint-niklaas.be/

I'll fix the Leopold II laan, this file is created programmatically so the PHP script will need fixing as well

Reply:
I mean this one: `Leopold Ii Laan 2,9100,Sint-niklaas`. @Bertware

Reply:
Got it!